src/components/Synthetics/TradeBox/hooks/useSummaryExecutionFee.ts

```diff
@@ -0,0 +1,91 @@
+import { useTokensData } from "context/SyntheticsStateContext/hooks/globalsHooks";
+import { useTradeboxExecutionFee } from "context/SyntheticsStateContext/hooks/tradeboxHooks";
+import {
+  estimateExecuteDecreaseOrderGasLimit,
+  getExecutionFee,
+  useGasLimits,
+  useGasPrice,
+} from "domain/synthetics/fees";
+import { estimateOrderOraclePriceCount } from "domain/synthetics/fees/utils/estimateOraclePriceCount";
+import { DecreasePositionSwapType } from "domain/synthetics/orders";
+import { SidecarLimitOrderEntry, SidecarSlTpOrderEntry } from "domain/synthetics/sidecarOrders/types";
+import { useSidecarEntries } from "domain/synthetics/sidecarOrders/useSidecarEntries";
+import { convertToUsd } from "domain/synthetics/tokens";
+import { useChainId } from "lib/chains";
+import { getByKey } from "lib/objects";
+import { useCallback, useMemo } from "react";
+
+export const useSummaryExecutionFee = () => {
+  const executionFee = useTradeboxExecutionFee();
+  const sidecarEntries = useSidecarEntries();
+  const tokensData = useTokensData();
+
+  const { chainId } = useChainId();
+
+  const gasLimits = useGasLimits(chainId);
+  const gasPrice = useGasPrice(chainId);
+
+  const getOrderExecutionFee = useCallback(
+    (swapsCount: number, decreasePositionSwapType: DecreasePositionSwapType | undefined) => {
+      if (!gasLimits || !tokensData || gasPrice === undefined) return;
+
+      const estimatedGas = estimateExecuteDecreaseOrderGasLimit(gasLimits, {
+        decreaseSwapType: decreasePositionSwapType,
+        swapsCount: swapsCount ?? 0,
+      });
```

Comment:
Hmm, I just moved these hooks [from Confirmation Box](https://github.com/gmx-io/gmx-interface/blob/master/src/components/Synthetics/ConfirmationBox/ConfirmationBox.tsx#L513-L537). Now I'm not actually understand why does it happening

Reply:
As @onemicky told, this value required in case when user edits tpsl order and we should cover case if saved order execution fee was exceeded.

I renamed the hook to `useTPSLSummaryExecutionFee` to make clear what actually it does